backend/database_handler/migration/versions/99d3f1bc5a08_add_hash.py

```diff
@@ -0,0 +1,101 @@
+"""add_hash
+
+Revision ID: 99d3f1bc5a08
+Revises: f9636f013003
+Create Date: 2024-09-05 17:03:30.743557
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "99d3f1bc5a08"
+down_revision: Union[str, None] = "f9636f013003"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+# This migration adds a hash column to the transactions table (instead of id) and a transaction_hash column to the transactions_audit table (instead of transaction_id).
+# To remain backwards compatible and downgradable, the hash and nonce are set to the current id on existing transactions.
```

Comment:
nit: in the start of the file there's a section for these kinds of comments

Reply:
gotcha